pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "python-kasa"
-version = "0.7.0.2"
+version = "0.7.0.3"
```

Comment:
Version bump should be done on its own PR.

Reply:
Done